### PR TITLE
Abracadabra

### PR DESCRIPTION
**Pull request by tannishmango**
update to schema 1.2.0
add new chains:
- arbitrum
- avalanche
- bnb
- fantom
- mainnet

### CODE REVIEW

File: subgraphs/abracadabra/config/bnb.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "startBlock": "12750809",
+  "network": "bnb",
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/config/bnb.json:3]: does this actually work? Shouldn't it be `bsc`?

File: subgraphs/abracadabra/src/common/constants.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export namespace Network {
   export const BSC = "BINANCE_SMART_CHAIN";
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/common/constants.ts:13]: You should update this file with the latest in the reference subgraph

File: subgraphs/abracadabra/config/bnb.json (hunk 1 above)
Comment (tannishmango) [reply subgraphs/abracadabra/config/bnb.json]: just updated this

File: subgraphs/abracadabra/src/common/getters.ts [hunk 3]
```diff
@@ -205,7 +205,7 @@ export function getOrCreateLendingProtocol(): LendingProtocol {
   LendingProtocolEntity.subgraphVersion = "0.0.6";
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/common/getters.ts:205]: subgraph version should be at least 1.0.0 since it's going to be deployed now

**Review by this-username-is-taken — DISMISSED** [review]

File: subgraphs/abracadabra/src/common/getters.ts (hunk 3 above)
Comment (tannishmango) [reply subgraphs/abracadabra/src/common/getters.ts]: fixed

**Pull request merged**